internal/controller/operatorconfigmap_controller.go

```diff
@@ -296,8 +321,13 @@ func (c *OperatorConfigMapReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			}
 		}
 
-		if err := c.reconcileClientOperatorSubscription(); err != nil {
-			c.log.Error(err, "unable to reconcile client operator subscription")
+		if err := c.reconcileSubscriptionLabelForWebhook(); err != nil {
+			c.log.Error(err, "unable to reconcile subscription label")
+			return ctrl.Result{}, err
+		}
+
+		if err := c.reconcileSubscriptions(); err != nil {
+			c.log.Error(err, "unable to reconcile subscriptions")
 			return ctrl.Result{}, err
```

Comment:
We should not bundle all subscription under a single reconcile function. There is a future part where we would like to parallaize the reconcile function and in that case you want each one of these to be its own thing and to do its own steps. If you need or want to refactor shared code, use a util function and call it from each of the specific reconcile functions